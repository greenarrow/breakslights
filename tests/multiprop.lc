MA01
MR01
A00 CFff0000 FDff ODff
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
