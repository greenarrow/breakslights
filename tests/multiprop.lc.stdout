00 000000 ff0000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
00 000000 000000 ff0000 ff0000 000000 000000 000000 000000 000000 000000 000000 000000
00 000000 000000 000000 ff0000 ff0000 ff0000 000000 000000 000000 000000 000000 000000
00 000000 000000 000000 000000 ff0000 ff0000 ff0000 ff0000 000000 000000 000000 000000
00 000000 000000 000000 000000 000000 ff0000 ff0000 ff0000 ff0000 ff0000 000000 000000
00 000000 000000 000000 000000 000000 000000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000
00 ff0000 ff0000 000000 000000 000000 000000 000000 ff0000 ff0000 ff0000 ff0000 ff0000
00 ff0000 ff0000 ff0000 ff0000 000000 000000 000000 000000 ff0000 ff0000 ff0000 ff0000
00 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 000000 000000 000000 ff0000 ff0000 ff0000
00 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 000000 000000 ff0000 ff0000
00 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 000000 ff0000
00 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000
00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
00 000000 ff0000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
